lib/grpc/GrpcService.ts

```diff
@@ -89,15 +78,30 @@ class GrpcService {
   }
 
   /**
-   * See [[Service.tokenSwap]]
+   * See [[Service.executeSwap]]
    */
-  public tokenSwap: grpc.handleUnaryCall<{ target_address: string, payload: TokenSwapPayload, identifier: string }, {}> = async (call, callback) => {
-    this.unaryCall(call.request, callback, this.service.tokenSwap);
+  public executeSwap: grpc.handleUnaryCall<{ target_address: string, payload: TokenSwapPayload, identifier: string }, {}> =
+  async (call, callback) => {
+    this.unaryCall(call.request, callback, this.service.executeSwap);
   }
 
   public shutdown: grpc.handleUnaryCall<{}, {}> = async (call, callback) => {
     this.unaryCall(call.request, callback, this.service.shutdown);
   }
+
+  /*
+   * See [[Service.subscribePeerOrders]]
+   */
+  public subscribePeerOrders: grpc.handleServerStreamingCall<{}, {}> = async (call) => {
+    this.service.on('peerOrder', (order: orders.PeerOrder) => call.write({ order }));
```

Comment:
I couldn't test it, but it seems that it will never trigger, because you didn't call `service.subscribePeerOrders`, so it will that it will subscribe to `pool`. 

but anyway I think we should use callbacks instead, as I wrote in a different comment.